src/System.Private.ServiceModel/tests/Common/Infrastructure/src/BridgeClient.cs

```diff
@@ -83,7 +85,7 @@ public static string GetResourceAddress(string resourceName)
             {
```

Comment:
Nit: consider a lock around the code block to prevent concurrent requests to Bridge.  Not critical but would reduce Bridge noise.